moveit_core/trajectory_processing/src/iterative_time_parameterization.cpp

```diff
@@ -35,6 +35,7 @@
 /* Author: Ken Anderson */
```

Comment:
Given your approach is still experimental, I think we should not change Ken Anderson's itarative_time_parameterization version but rather add a secondary approach that can be optionally changed to (and documented). So I think you should copy this file and call it something different. You will also need to copy the [planning request adaptor plugin](https://github.com/ros-planning/moveit/blob/kinetic-devel/moveit_ros/planning/planning_request_adapter_plugins/src/add_time_parameterization.cpp) and name it something different. This will ensure we don't change the behavior of currently deployed systems.